packages/react-router/src/Asset.tsx

```
@@ -1,5 +1,6 @@
 import * as React from 'react'
 import type { RouterManagedTag } from '@tanstack/router-core'
+import { useRouter } from './useRouter'
```

ESLint

[error] `./useRouter` import should occur before type import of `@tanstack/router-core`

(import/order)